README.md

```diff
@@ -23,19 +23,46 @@ If the output is correct, the entire configuration is cleared.
 If any check fails, the configuration retains the failed check at the top of the `<k>` cell.
 
 [ethereum.md](ethereum.md) is also capable of executing tests of arbitrary IELE transactions (subject to the fact that this first release of IELE is still
-built on top of an ethereum-like network layer). There are several differences in the syntax of the JSON test suite compared to the EVM test suite:
-
-* Users can specify the function called and the arguments to the function in blockchain tests. These fields are not currently part of the signed portion of the transaction,
-  for ease of compatibility with EVM tests. Transaction security will come in a future release.
-* Users can ask for gas checks in the test to be disabled by adding a `"checkGas": false` entry to the top-level JSON object.
-* VM tests call the function named `deposit`.
-* Users can specify a filename instead of a hexadecimal string when specifying the code of a contract, and the code will be pulled by assembling the file, which contains
-  IELE assembly. Filenames are relative to the directory containing the JSON file.
-* Blockchain tests have only one possible schedule, `Albe`.
-* Many fields that are checked by the Ethereum tests can be omitted or given arbitrary values, including:
-    * `blockHeader` does not check any values aside from those which are input to the VM, and the remaining can be given arbitrary values.
-    * `data` in a transaction can be an arbitrary value, but it still must match the signature of the transaction.
-    * `genesisBlockHeader` and `rlp` can be omitted.
+built on top of an ethereum-like network layer).
```

Comment:
It may still be worthwhile saying that the format resembles the EVM test suite format for people familiar with it.